### DISCUSSION TITLE
Fetch latest version numer

### DISCUSSION THREAD

**Discussion by dapuzz** [mikebrady/shairport-sync#1598] [Q&A]
Hello
Why latest tag here is the 4.1 released on Oct 28 instead of 4.1.1 released 2 weeks ago?
https://api.github.com/repos/mikebrady/shairport-sync/releases/latest

Is there any way to get latest version number?
Thank you

**Comment by mikebrady** [comment]
Thanks for the post -- I'll have to look into this.

**Comment by mikebrady** [comment]
Thanks again for this. Seems to be a newish feature -- one that I missed -- to be able to [explicitly set the latest release](https://github.blog/changelog/2022-10-21-explicitly-set-the-latest-release/).
